.eslintrc.cjs

```diff
@@ -34,13 +37,14 @@ module.exports = {
       },
     ],
     '@typescript-eslint/consistent-type-definitions': 'warn',
-    '@typescript-eslint/no-unsafe-assignment': 'warn',
+    '@typescript-eslint/no-unsafe-assignment': 'off',
     '@typescript-eslint/no-unsafe-call': 'warn',
     '@typescript-eslint/no-unsafe-member-access': 'warn',
     '@typescript-eslint/no-non-null-assertion': 'warn',
     '@typescript-eslint/no-unnecessary-condition': 'warn',
     '@typescript-eslint/ban-types': 'warn',
     '@typescript-eslint/no-empty-interface': 'warn',
     '@typescript-eslint/no-unused-vars': 'warn',
+    '@typescript-eslint/no-empty-function': 'warn',
```

Comment:
we shouldn't be changing linting rules unless there is a good reason to. once linting is set, it's like the standard the team needs to obey by.

Reply:
I agree,

To give you context, the previous rules had issues and that is why @nam-m , needed to temporary fix it here.

@nam-m , make sure to pull the latest into your branch and also ensure you remove any linting changes.

Reply:
Gotcha, will update with latest lint rules